loris/resolver.py

```diff
@@ -61,13 +65,40 @@ def resolve(self, ident):
             ident (str):
                 The identifier for the image.
         Returns:
-            (str, str): (fp, format)
+            ImageInfo: Partially constructed ImageInfo object 
```

Comment:
fixed the issues. The tiff content that was being passed for the resolver tests wasn't valid for Pillow's code.